desktop/apps/auction/routes.js

```diff
@@ -47,12 +47,23 @@ export async function index (req, res, next) {
 
     // TODO: Refactor out Backbone
     const auctionModel = new Auction(sale)
+    const isEcommerceSale = saleId === 'chris-ecommerce-sale'
```

Comment:
FIXME: This needs to be addressed after https://github.com/artsy/auctions/issues/683#issuecomment-359154280 is answered